julia/Project.toml

```diff
@@ -1,15 +1,8 @@
-name = "TP-IV Exercises"
-authors = ["Matteo Seclì <matteo.secli@epfl.ch>",
-           "Alberto Mercurio <alberto.mercurio@epfl.ch>"]
-
 [deps]
-Arpack = "7d9fca2a-8960-54d3-9f78-7d1dccf2cb97"
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
-DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
 IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
-KrylovKit = "0b1a1467-8014-51b9-945f-bf0ae24f4b77"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
```

Comment:
I'd keep this as it's a direct dependency.

Reply:
LinearAlgebra is a standard library, so we don't need it

Reply:
For KrylovKit, we already have Krylov subspace stuff inside QuantumToolbox.jl